### PR TITLE
Daniel

### PR DESCRIPTION
**Pull request by dezbrid**
## Summary

> 

- Install and  initial settings to Eslint package
- Change function  in  the file index  for 01-spread and 02-promises , and execute test

## Screenshots
imagen
imagen
imagen

## Trello Card
https://trello.com/c/ba7RB0M1/6-n-37-js-stack-iii-ecma-6-7

### CODE REVIEW

File: finger-exercises/ecmascript/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,14629 @@
+{
+  "name": "ecmascript",
```
Comment (sfernandez11) [inline finger-exercises/ecmascript/package-lock.json]: Delete this file, we use `yarn`, not `npm`

File: finger-exercises/ecmascript/02-promises/index.js [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+      resolve(time);
+    }
+  });
 }
 
-export function asyncDelay() {
-
+export async function asyncDelay(timeout) {
+  return await timeout;
 }
```
Comment (sfernandez11) [inline finger-exercises/ecmascript/02-promises/index.js]: You can use: `export const asyncDelay = delay;` directly

File: finger-exercises/ecmascript/02-promises/index.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  return new Promise(function (resolve, reject) {
+    if (time > 500) {
+      reject(new Error("This time is too much !"));
+    } else {
+      resolve(time);
```
Comment (sfernandez11) [inline finger-exercises/ecmascript/02-promises/index.js]: Here you must return the `elapsedTime` between now and now + time after a `setTimeout`. In this `delay` you only return a `time` but what’s happened with the "delay" simulation?

File: finger-exercises/ecmascript/.eslintrc.js [hunk 4]
```diff
@@ -1,10 +1,24 @@
+// eslint-disable-next-line no-undef
 module.exports = {
```
Comment (sfernandez11) [inline finger-exercises/ecmascript/.eslintrc.js:2]: Why changes in this file? What is the problem?

**Review by sfernandez11 — CHANGES_REQUESTED** [review]

**Comment by sfernandez11** [comment]
Good work, some comments!

File: finger-exercises/ecmascript/package-lock.json (hunk 1 above)
Comment (dezbrid) [reply finger-exercises/ecmascript/package-lock.json]: ok, i will delete this file and  re install with yarn

**Pull request closed without merging**